### PR TITLE
DMI preview in VScode

### PR DESCRIPTION
**Pull request by spookydonut**
aaa

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
 	"eslint.workingDirectories": [
         "./tgui"
-    ]
+	],
+	"workbench.editorAssociations": [
```
Comment (ZeWaka) [inline .vscode/settings.json:5]: No comment explaining this? Shameful.

Comment (steamp0rt) [reply .vscode/settings.json:5]: goddammit

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {
 	"eslint.workingDirectories": [
```
Comment (ZeWaka) [inline .vscode/settings.json:2]:
```suggestion
{

	// Sets the working directory for the eslint extension
	"eslint.workingDirectories": [
```

File: .vscode/settings.json (hunk 1 above)
Comment (ZeWaka) [inline .vscode/settings.json:5]:
```suggestion
	],

	// Lets you preview .dmi files in the editor
	"workbench.editorAssociations": [
```

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Comment by TheGamerdk** [comment]
JSON doesn't have comments unless I'm missing something?

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
 {
+
+	// Sets the working directory for the eslint extension
 	"eslint.workingDirectories": [
-        "./tgui"
-    ]
+		"./tgui"
+	],
+
+	// Lets you preview .dmi files in the editor
```
Comment (spookydonut) [inline .vscode/settings.json:8]:
```suggestion
```

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,5 +1,15 @@
 {
+
+	// Sets the working directory for the eslint extension
```
Comment (spookydonut) [inline .vscode/settings.json:3]:
```suggestion
```

**Review by SpaceManiac — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**